api/candlepin-api-spec.yaml

```diff
@@ -8439,8 +8439,6 @@ components:
                 $ref: '#/components/schemas/ReleaseVerDTO'
               owner:
                 $ref: '#/components/schemas/NestedOwnerDTO'
-              environment:
```

Comment:
Are we able to remove this at this point? I think we'll need to support both `environment` and `environments` for some time due to older clients that won't be aware of the new multi-environment functionality.

Reply:
Just checked the design doc: this needs to be retained, and the output needs to be adjusted to be a comma-delimited list of environment names.

Reply:
yep. we should retain the field here, but the output adjustment will be in one of the other task in the epic

Reply:
Retained.